DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/projectingUsingFunctions/projectingUsingFunctions.java

```diff
@@ -0,0 +1,48 @@
+//region Usings
+import net.ravendb.client.documents.session.IDocumentSession;
+import net.ravendb.client.documents.session.IRawDocumentQuery;
+//endregion
+import net.ravendb.demo.common.DocumentStoreHolder;
+
+
+
+import java.util.List;
+
+public class projectingUsingFunctions {
+
+    //region Demo
+    public static class Employee {
```

Comment:
In the C# demo, this class is called `EmployeeDetails`

Reply:
Fixed